Assets/Scripts/Models/Character/Character.cs

```diff
@@ -577,4 +582,18 @@ private void LoadStats()
 
         Debug.ULogChannel("Character", "Initialized " + stats.Count + " Stats.");
     }
+
+    /// <summary>
+    /// Use the stats of the character to determine various traits.
+    /// </summary>
+    private void UseStats()
+    {
+        // Base character speed on their dexterity.
+        speed = Convert.ToSingle(stats["Dexterity"].Value) / 2;
+
```

Comment:
Perhaps speed should be calculated on the fly so if something later becomes capable of changing stats, speed will also be affected, or perhaps a better route would be to have a function that sets stats, so when it is used and stats are changed they can be properly updated (which for maxHP we probably wouldn't also want to set currentHP, since then crew would be healed when their con changes)